packages/destination-actions/src/destinations/index.ts

```diff
@@ -23,6 +23,7 @@ export const manifest: Record<MetadataId, ManifestEntry> = {}
  * in production and sync them into staging via `sprout`.
  */
 register('60f64ae3eaebd66d17d28e9f', './1plusx')
+register('6127d2a468778ec5f0023103', './actions-facebook-conversions')
```

Comment:
Wonder if this should be

```suggestion
register('6127d2a468778ec5f0023103', './facebook-conversions-api')
```

to match other destinations and name of integration.

Reply:
It doesn't really matter tbh. Convention is that the directory names do not have the redundant `actions-` part that is found in most of these destinations' slugs.